### PR TITLE
VTAdmin: Upgrade msw package to address security vuln

### PR DESCRIPTION
**Pull request by mattlord**
## Description

Upgrade the node `msw` package to address security vuln in its `cookie` pkg dep: https://github.com/vitessio/vitess/security/dependabot/395

## Related Issue(s)
  
  - Fixes: https://github.com/vitessio/vitess/security/dependabot/395

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [x] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on CI?
-   [x] Documentation was added or is not required

### CODE REVIEW

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Looks like the column name casing fix also got checked in by mistake.

**Review by GrahamCampbell — COMMENTED** [review]
There are lots of breaking changes to navigate: https://mswjs.io/docs/migrations/1.x-to-2.x.

**Review by notfelineit — APPROVED** [review]

File: web/vtadmin/src/components/ActionPanel.test.tsx [hunk 1]
```diff
@@ -33,11 +33,7 @@ describe('ActionPanel', () => {
      * provides such a function and should be `render`ed in the context QueryClientProvider.
      */
     const Wrapper: React.FC<ActionPanelProps & { url: string }> = (props) => {
-        const mutation = useMutation(() => fetch(new URL(props['url']), { method: 'post' }), {
-            onError: (error) => {
-                console.log('ERROR: ', error);
-            },
-        });
+        const mutation = useMutation(() => fetch(new URL(props['url']), { method: 'post' }));
```
Comment (frouioui) [inline web/vtadmin/src/components/ActionPanel.test.tsx:36]: Was the removal of the console.log for the error intentional? I am guessing this only impacts our own debugging on vtadmin.

**Review by frouioui — APPROVED** [review]

File: web/vtadmin/src/components/ActionPanel.test.tsx (hunk 1 above)
Comment (notfelineit) [reply web/vtadmin/src/components/ActionPanel.test.tsx:36]: Yeah - I removed it because I accidentally left it in there!

**Pull request merged**